<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloaderDialog</class>
 <widget class="QDialog" name="DownloaderDialog">
  <property name="windowTitle">
   <string>Download OSL Dataset from HuggingFace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="apiKeyLayout">
     <item>
      <widget class="QLabel" name="labelApiKey">
       <property name="text">
        <string>HuggingFace API Key:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditApiKey">
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="urlLayout">
     <item>
      <widget class="QLabel" name="labelUrl">
       <property name="text">
        <string>OSL JSON URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUrl"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="outputDirLayout">
     <item>
      <widget class="QLabel" name="labelOutputDir">
       <property name="text">
        <string>Output Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditOutputDir">
       <property name="text">
        <string>downloaded_data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxDryRun">
     <property name="text">
      <string>Dry run (only estimate required space, do not download)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar"/>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditLog">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="maximumHeight">
      <number>150</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="pushButtonDownload">
       <property name="text">
        <string>Start Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="text">
        <string>Exit Tool</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
